packages/design-system/src/components/N8nPopover/index.ts

```diff
@@ -0,0 +1,3 @@
+import N8nPopover from './Tooltip.vue';
```

Comment:
nitpick - maybe rename Tooltip.vue to Popover.vue for consistency

Reply:
Done! Thanks for pointing it out.